protocol/synthetix/contracts/modules/core/MarketManagerModule.sol

```diff
@@ -139,7 +139,8 @@ contract MarketManagerModule is IMarketManagerModule {
         if (msg.sender != marketData.marketAddress) revert AccessError.Unauthorized(msg.sender);
 
         // Ensure that the market's balance allows for this withdrawal.
-        if (amount > getWithdrawableUsd(marketId)) revert NotEnoughLiquidity(marketId, amount);
+        if (amount > getWithdrawableMarketUsd(marketId))
```

Comment:
in every case in the codebase tmk we put curly braces around all if statements even if they are only 1 line long